plugins/utils/comment-nvim.nix

```diff
@@ -48,7 +48,7 @@ in {
         default = null;
       };
       opleader = mkOption {
-        type = types.nullOr (types.submodule ({...}: {
+        type = types.nullOr (types.submodule (_: {
```

Comment:
Better to use `helpers.mkCompositeOption` here.

Reply:
This plugins isn't utilizing helper functions from what I can see. Maybe we'll need a separate PR for refactoring this plugin